File: Backend/barbershop/barbershop/Models/ResponseDTOs/UserDTO.cs
using System;

namespace barbershop.Models.ResponseDTOs
{
    public class UserDTO
    {
        public long? UserId { get; set; }
        public string? Email { get; set; }
        public string? UserName { get; set; }
        public string? Password { get; set; }
        public string? FullName { get; set; }
        public string? Phone { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string? Address { get; set; }
        public string? CCCD { get; set; }
        public int? RoleId { get; set; }
        public string? AvatarUrl { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public bool? IsActive { get; set; }
    }
}

File: Frontend/barbershop/src/types/ResponseDTOs/userDTO.ts
// Bạn có thể đặt ở: f:\MyProject\BarberShop\Frontend\barbershop\src\types\UserDTO.ts
import type { LoyaltyPointDTO } from './loyaltyPointDTO';
export type UserDTO = {
    userId?: number;
    email?: string;
    userName?: string;
    password?: string;
    fullName?: string;
    phone?: string;
    dateOfBirth: Date;
    address?: string;
    cccd?: string;
    roleId?: number;
    createdAt?: Date;
    updatedAt?: Date;
    avatarUrl?: string;
    isActive?: boolean;
    loyaltyPointDTO?: LoyaltyPointDTO;
};
